README.md

```diff
@@ -1,18 +1,8 @@
-# svelte app
+# svelte playground
 
-This is a project template for [Svelte](https://svelte.technology) apps. It lives at https://github.com/sveltejs/template-webpack.
-
-To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
-
-```bash
-npm install -g degit # you only need to do this once
-
-degit sveltejs/template-webpack svelte-app
-cd svelte-app
-```
-
-*Note that you will need to have [Node.js](https://nodejs.org) installed.*
+This project was created with the goal of gaining practical experience and insight from working with web components and the Svelte UI framework.
 
+It was built using a template for [Svelte](https://svelte.technology) apps. The original template lives at https://github.com/sveltejs/template-webpack.
 
 ## Get started
```

Comment:
How do you feel about breaking this up into 2 sections: **Initial Setup** and **Development**?

Initial Setup
- clone the repo
- cd into it
- install dependencies

Development
- we recommend `vs code`, npm version, node version, etc.
- use `npm run dev` script

We may eventually also have a Deployment instructions in the future.

Reply:
Sounds good. 

Recommending `vs code` is not necessary; editor defaults should be addressed and accounted for with proper linting setup.

We can address node/npm version later when we have a chance to discuss. 

Also, I think we should consider removing existing deployment instructions as part of #9 (Remove serve dependency).